waf_cgo_disabled.go

```diff
@@ -0,0 +1,13 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Unsupported Go versions (>=)
```

Comment:
```suggestion
// The Go build tag "appsec" was introduced to avoid having CGO_ENABLED=0 breaking changes
// due to purego's dynamic link against libdl.so, which is not expected when CGO is disabled. 
```